pkgs/tools/networking/gping/default.nix

```diff
@@ -1,32 +1,25 @@
 { lib
-, iputils
-, python3
-, python3Packages
+, rustPlatform
+, fetchFromGitHub
 }:
 
-python3Packages.buildPythonApplication rec {
+rustPlatform.buildRustPackage rec {
   pname = "gping";
-  version = "1.1";
+  version = "unstable-2020-11-14";
```

Comment:
I think it is preferable to have the release version. It builds just fine without the `LICENSE` file and we do not use the license file anyway?

Reply:
Totally agree, just wasn’t sure if it was necessary for distribution.